app/admin/settings/Settings.scala

```diff
@@ -57,13 +47,14 @@ object Settings {
     } yield settings
 }
 
-case class SettingsSources(switches: SettingsSource)
+case class SettingsSources(switches: SettingsSource, amounts: SettingsSource)
 
 object SettingsSources {
   def fromConfig(config: Config): Either[Throwable, SettingsSources] = {
     for {
       switchesSource <- SettingsSource.fromConfig(config, "switches")
-    } yield SettingsSources(switchesSource)
+      amountsSource <- SettingsSource.fromConfig(config, "amounts")
```

Comment:
is this reloaded regularly while the app is running or would it only run on startup?  obviously doesn't matter at this stage, but I'm interested.

Reply:
The `SettingsSource` is loaded once at startup - it tells the program where to find the amounts config. The switches/amounts config *is* then reloaded regularly